packages/gestalt/src/DropdownContext.js

```diff
@@ -8,15 +8,18 @@ type DropdownContextType = {|
   setOptionRef: (?HTMLElement) => void,
 |};
 
-const initialState = {
+const initialContextState = {
   id: '',
   hoveredItem: -1,
   setHoveredItem: () => {},
   setOptionRef: () => {},
 };
 
-const DropdownContext: Context<DropdownContextType> = createContext<DropdownContextType>(
-  initialState,
+const context: Context<DropdownContextType> = createContext<DropdownContextType>(
+  initialContextState,
 );
 
-export default DropdownContext;
+const DropdownContextProvider = context.Provider;
+const DropdownContextConsumer = context.Consumer;
+
+export { DropdownContextProvider, DropdownContextConsumer };
```

Comment:
renamed to standarize context files. We were exporting context rather than provider. now we export both provider and consumer so when we read the import at the top we now what's actually happening rather than having to read the line to know if we are setting context or consuming.